Short-Answer/Algorithms_Answers.md

```diff
@@ -1,7 +1,19 @@
 # Answers to Short Answer Questions
 
-## A: The first one appears to be linear, since the while loop will get bigger depending on what N is, the computations will not increase
+A: The first one appears to be linear, since the while loop will get bigger depending on what N is, the computations will not increase
 
-## B: This appears to be O(N^2) since it has multiple nested loops and the computations will exponentially grow depending on how large n is. This is not efficient
+B: This appears to be O(N^2) since it has multiple nested loops and the computations will exponentially grow depending on how large n is. This is not efficient
 
-## C: This is going to be 2^n since this appears to be the Fibonacci sequence, and the number of calls will increase as n increases since this is a recursive function.
+C: This is going to be 2^n since this appears to be the Fibonacci sequence, and the number of calls will increase as n increases since this is a recursive function
```

Comment:
Good job breaking down your explanations Michael. Problem C ends up being `O(n)`, and Exercise 2 we talked about how 'word problems' can be a little confusing. If you have time to revisit think about the `building` as a `list/array` and each `floor` as an `index`, looking at it this way may help you dial in on a good algorithm for that scenario 👍